tests/test_google_sheets_datatypes.py

```diff
@@ -241,8 +241,8 @@ def test_run(self):
 
                             # BUG_TDL-14448 | https://jira.talendforge.org/browse/TDL-14448
                             #                 Skipping Number and Currency columns with boolean values because they do not fallback to string
-                            elif test_case == 'boolean' and column in {'Currency', 'Number'}: # BUG_TDL-14448
-                                continue  # skip
+                            # elif test_case == 'boolean' and column in {'Currency', 'Number'}: # BUG_TDL-14448
+                            #     continue  # skip
```

Comment:
If we have resolved this issue then we should remove this section rather than commenting it.

Reply:
Removed the commented code